.eslintrc.js

```diff
@@ -18,6 +18,6 @@ module.exports = {
   // add your custom rules here
   rules: {
     // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off' //conditional statement to no-debugger variable
```

Comment:
I think this comment is redundant - it simply re-iterates what is already written in code.